### ISSUE TITLE
автоматизация blockcheck.sh и install_easy.sh

### ISSUE THREAD

**Issue by kotsmotritnastul** [bol-van/zapret#556]
стали очень часто меняться настройки на ТСПУ. может ли кто-нибудь сделать башскрипт который будет делать следующее:

1) останавливает запрет
2) запускает блокчек с сохранёнными настройками (адрес ггц, количество попыток, быстрый или стандартный тест и т.д.)
3) сохраняет предложенные варианты обхода блокировок в переменные или в текстовый файл
4) запускает ./install_easy.sh
5) подсовывает в него сохранённые параметры обхода блокировок
6) нажимает ввод во всех остальных случаях и запускает блокчек

такой скрипт можно было бы положить в кронтаб и запускать пока на работе работаешь, было бы очень удобно - сейчас после работы пара часов есть, и полчаса-час уходит на прогон блокчека потому что в рабочие часы они что-то опять поменяли. я пытался накостылить сам такое решение, но для этого надо менять имеющиеся баш-скрипты а они очень большие и я в них запутался.

**Comment by bol-van** [comment]
Здесь надо уходить от копипасты, записывать стратегии и находить общий знаменатель.
Может проблема банально в мешяющемся ttl

**Comment by kotsmotritnastul** [comment]
> Здесь надо уходить от копипасты, записывать стратегии и находить общий знаменатель. Может проблема банально в мешяющемся ttl

нет, меняются методы типа split/split2/fake и т.д. и кроме того на данный момент это всё работает, просто это надо делать руками, а это долго и муторно, возможно кто-нибудь возьмётся это автоматизировать.

**Comment by kotsmotritnastul** [comment]
> А можно вообще хотя бы узнать, что за параметры выдает блокчеккер и что с ними делать дальше? Потому что я не понимаю абсолютно НИЧЕГО, а в мануале про этот момент так же написано НИЧЕГО

я в своё время написал для себя и других памятку пошаговую https://gist.github.com/kotsmotritnastul/3cf8296a14d02bb95961c093082f41c0 люди вроде осиливают, может и тебе пригодится.

**Comment by MakaiHanta** [comment]
> > А можно вообще хотя бы узнать, что за параметры выдает блокчеккер и что с ними делать дальше? Потому что я не понимаю абсолютно НИЧЕГО, а в мануале про этот момент так же написано НИЧЕГО
> 
> я в своё время написал для себя и других памятку пошаговую https://gist.github.com/kotsmotritnastul/3cf8296a14d02bb95961c093082f41c0 люди вроде осиливают, может и тебе пригодится.

Все равно ничего не понятно. Почему ты одни параметры записываешь в одну строку, а другие в другую? Я вот при подготовке блокчеккера везде энтер нажимал и мне выдало 4 строки в саммери. Как понять, какой параметр куда вписывать?

**Comment by kotsmotritnastul** [comment]
> Все равно ничего не понятно. Почему ты одни параметры записываешь в одну строку, а другие в другую? Я вот при подготовке блокчеккера везде энтер нажимал и мне выдало 4 строки в саммери. Как понять, какой параметр куда вписывать?

так в моей пошаговой инструкции же написано что куда пихать - параметр `curl_test_https_tls12` из блокчека относится к строке `NFQWS_OPT_DESYNC_HTTPS` в конфиге, а параметр `curl_test_http3` из блокчека относится к строке `NFQWS_OPT_DESYNC_QUIC` в конфиге. делай пошагово, по ней как минимум несколько десятков невдупляющих страдальцев делали и всё получилось.

**Comment by MakaiHanta** [comment]
> > Все равно ничего не понятно. Почему ты одни параметры записываешь в одну строку, а другие в другую? Я вот при подготовке блокчеккера везде энтер нажимал и мне выдало 4 строки в саммери. Как понять, какой параметр куда вписывать?
> 
> так в моей пошаговой инструкции же написано что куда пихать - параметр `curl_test_https_tls12` из блокчека относится к строке `NFQWS_OPT_DESYNC_HTTPS` в конфиге, а параметр `curl_test_http3` из блокчека относится к строке `NFQWS_OPT_DESYNC_QUIC` в конфиге. делай пошагово, по ней как минимум несколько десятков невдупляющих страдальцев делали и всё получилось.

Блин, я бы с радостью, но я тут другую проблему обноружил. У меня дискорд через браузер работает вообще без каких-либо нареканий или способов обхода в самом браузере, но вот флатпак версия приложения не запускается. И как мне в таких условиях вообще настроить эту прогу?
